### PR TITLE
Move the "parent" object to be the first member of each GATT type.

### PR DESCRIPTION
**Pull request by jyasskin**
Demo at https://rawgit.com/jyasskin/web-bluetooth-1/parent-order/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2089,8 +2089,8 @@
 
         <pre class="idl">
           interface BluetoothGATTDescriptor {
-            readonly attribute UUID uuid;
             readonly attribute BluetoothGATTCharacteristic characteristic;
```
Comment (scheib) [inline index.html]: Move note section below as well.

File: index.html [hunk 2]
```diff
@@ -1681,8 +1681,8 @@
 
         <pre class="idl">
           interface BluetoothGATTCharacteristic : CharacteristicEventHandlers {
-            readonly attribute UUID uuid;
             readonly attribute BluetoothGATTService service;
+            readonly attribute UUID uuid;
```
Comment (scheib) [inline index.html]: Update note section ordering as well.

File: index.html [hunk 3]
```diff
@@ -1516,9 +1516,9 @@
 
         <pre class="idl">
           interface BluetoothGATTService : ServiceEventHandlers {
+            readonly attribute BluetoothDevice device;
```
Comment (scheib) [inline index.html]: Update note section ordering as well.

File: index.html (hunk 1 above)
Comment (jyasskin) [inline index.html]: Done, thanks.

File: index.html (hunk 2 above)
Comment (jyasskin) [inline index.html]: Done.

File: index.html (hunk 3 above)
Comment (jyasskin) [inline index.html]: Done.

**Comment by scheib** [comment]
LGTM

**Pull request merged**